### DISCUSSION TITLE
studenten-aan-het-woord

### DISCUSSION THREAD

**Discussion by Marwaxhello** [fdnd/docs.fdnd.nl#83] [Reflectie]
**Welke onderwerpen verdienen nog aandacht en zijn hierboven niet aangestipt?**

- Ik zou graag meerdere introductielessen JavaScript willen volgen, aangezien ik momenteel een beperkte kennis heb van JavaScript. Dit is noodzakelijk voor mijn stage, waar dit nu van mij wordt verwacht, terwijl ik pas net begin met het leren ervan.
- Hoewel onze beoordeling heel relaxed en duidelijk is en je niet uit een boek hoeft te leren, is het toch gebaseerd op praktijk en beoordeling. Toch vind ik het soms bevooroordeeld door de beoordeling van leraren, omdat de ene dit leuk vindt en de andere niet.
- Het zou ook fijn zijn als de lessen worden gefilmd voor mensen die niet aanwezig kunnen zijn of die ze later willen terugkijken.

**Wat vertel jij over je opleiding op een feestje?**

- Onze assessments zijn echt SUPER chill, we krijgen bijvoorbeeld 15 punten per assessment.
- We hebben zelfs een pingpongtafel, een relaxte locatie om te chillen.
- Je kunt een meer persoonlijke band opbouwen met je docenten.
- Het is een tweejarige opleiding.
- We hanteren het motto WORK HARD PLAY HARD aan het einde van elke sprint.
- Er worden automatisch evenementen georganiseerd om in contact te komen met het bedrijfsleven.
- Hier kun je zijn wie je wilt zijn.

**Comment by ju5tu5** [comment]
Bijna alles opgenomen in de tekst.
